Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+                PatternBinding(pattern: IdentifierPattern(identifier: .identifier("mySecondLet")),
+                               typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("CGFloat")))
+            })
+
+            MemberDeclListItem(decl: VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                    PatternBinding(pattern: IdentifierPattern(identifier: .identifier("myOtherVar")),
+                                   typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("String")))
+                })
+            )
+
+            StructDecl(identifier: "InnerStruct", members: MemberDeclBlock())
+        })
+        )
+
+        let syntax = myStruct.buildSyntax(format: Format())
+        XCTAssertEqual(syntax.description, """
+        struct MyStruct{
+            var myFirstVar: Int
+            let mySecondLet: CGFloat
+            let myOtherVar: String
+            struct InnerStruct{
+            }
+        }
+        """)
+    }
+
+    func testExpressibleAsCodeBlockItem() {
+        let myCodeBlock = SourceFile(eofToken: SyntaxFactory.makeToken(.eof, presence: .present)) {
+            StructDecl(identifier: "MyStruct1", members: MemberDeclBlock())
+
+            StructDecl(identifier: "MyStruct2", members: MemberDeclBlock())
+        }
+
+        let syntax = myCodeBlock.buildSyntax(format: Format())
+        XCTAssertEqual(syntax.description, """
+
+        struct MyStruct1{
+        }
+        struct MyStruct2{
+        }
+        """)
+    }
+
+    func testExpressibleAsSwitchStmt() {
+        let versions = [("version_1", "1.0.0"), ("version_2", "2.0.0"), ("version_3", "3.0.0"), ("version_3_1", "3.1.0")]
+        let expression = IdentifierExpr(identifier: SyntaxFactory.makeIdentifier("version"))
+
+        let switchStmt = SwitchStmt(labelName: nil,
+                                    expression: expression,
+                                    leftBrace: .leftBrace.withTrailingTrivia(.newlines(1)),
+                                    rightBrace: .rightBrace.withLeadingTrivia(.newlines(1)),
+                                    casesBuilder: {
+            for (version, semVer) in versions {
+                SwitchCase(label: SwitchCaseLabel(caseItemsBuilder: {
+                    CaseItem(pattern: EnumCasePattern(caseName: version))
+                }), statementsBuilder: {
+                    ReturnStmt(expression: StringLiteralExpr(semVer, closeQuote: .stringQuote.withTrailingTrivia(.newlines(1))))
+                })
+            }
+        })
+
+        let syntax = switchStmt.buildSyntax(format: Format())
+
+        XCTAssertEqual(syntax.description, """
+        switch version{
+        case .version_1:
+        return "1.0.0"
+        case .version_2:
+        return "2.0.0"
+        case .version_3:
+        return "3.0.0"
+        case .version_3_1:
+        return "3.1.0"
+
+        }
+        """)
+    }
+}
+
+extension ReturnStmt: ExpressibleAsCodeBlockItem {
```

Comment:
This should probably live in `SwiftSyntaxBuilder` and not the test suite, right?

Reply:
Oh yes. Sorry! 
Was playing around. I will also make Statements implement the `ExpressibleAsCodeBlockItem`. 
Then people can add switch case, return, if and so on.

Reply:
@ahoppen added `ExpressibleAsCodeBlockItem` protocol to `Stmt`